lib/utils/report.mjs

```diff
@@ -241,13 +239,12 @@ function isFixApplied({ fix, line, result: { stylelint }, ruleName }) {
  * @param {StylelintPostcssResult['fixersData']} o.fixersData
  * @param {string} o.ruleName
  * @param {Range} [o.range]
- * @param {boolean} o.fixed
  * @todo stylelint/stylelint#7192
```

Comment:
Are you planning to eventually pass the range to `addFixData`?
e.g.

```
const range = {
  start: fixedNode.source.start,
  end: fixedNode.source.end,
};

addFixData({ fixersData, ruleName, range });
```

So that we can close #4664 since we will be able to add the range+path to the fixed section of the `verbose` formatter's output.
i.e. click in the terminal, get to the line in your IDE and eventually scroll horizontally to reach the range

Reply:
See: https://github.com/stylelint/stylelint/issues/4664#issuecomment-2568437149